pages.ml/common/cargo.md

```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> റസ്റ്റ് പ്രോജക്റ്റുകളും അവയുടെ മൊഡ്യൂൾ ഡിപൻഡൻസികളും (ക്രാറ്റുകൾ) കൈകാര്യം ചെയ്യുക.
+> `ബിൽഡ്` പോലുള്ള ചില ഉപകമാൻഡുകൾക്ക് അവരുടേതായ ഉപയോഗ ഡോക്യുമെന്റേഷൻ ഉണ്ട്.
+> കൂടുതൽ വിവരങ്ങൾ: <https://doc.rust-lang.org/cargo>.
+
+- ക്രേയ്റ്റ്സ് തിരയുക:
+
+`cargo search {{ക്രേയ്‌റ്റ്സിന്റെ പേര്}}`
```

Comment:
```suggestion
`cargo search {{തിരയൽ സ്ട്രിംഗ്}}`
```
Should this for the search string placeholder, seems like you use the same one as the `{{cargo_name}}` placeholder.